### ISSUE TITLE
lxip: build error with gcc 10

### ISSUE THREAD

**Issue by cproc** [genodelabs/genode#4102]
When building `LIB=lxip` with gcc 10, the following error occurs:

```
  Library lxip
    MERGE    lxip.lib.so
genode-x86-ld: lxc_emul.o:/.../repos/dde_linux/src/lib/lxip/lxc_emul.c:459: multiple definition of `loopback_net_ops'; drivers/net/loopback.o:/.../contrib/dde_linux-fb5909112c26bcbc48cadae5ccf18d5546c6e86c/src/lib/lxip/drivers/net/loopback.c:220: first defined here
```

The error occurs with gcc 10 because it has the '-fno-common' option set by default.

**Comment by cproc** [comment]
Fixed in af798e4

**Comment by chelmuth** [comment]
Merged to staging.
